backend/src/main/java/com/google/capstone/TextToSpeech.java

```diff
@@ -49,10 +52,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.setContentType("text/plain");
         response.getWriter().println(blobKey);
       } catch (IOException e) {
-        response.sendError(500, "Unable to generate audio file.");
+        String errorMsg = "Unable to generate audio file.";
+        logger.severe(errorMsg);
```

Comment:
It seems that we can add more contexts in the logging message. Think about what information would help you when you are debugging. For example you can add the textString and objectIdString.

Reply:
changed